files/es/mdn/writing_guidelines/writing_style_guide/index.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Guía de estilo
+slug: MDN/Writing_guidelines/Writing_style_guide
+tags:
+  - Proyecto_MDC
+  - Todas_las_Categorías
+translation_of: MDN/Guidelines/Writing_style_guide
+original_slug: MDN/Guidelines/Writing_style_guide
+---
+{{MDNSidebar}}
+
+Para mostrar la documentación de forma organizada, estandarizada y fácil de leer, la guía de estilos de MDN describe cómo debe organizarse, escribirse y formatearse el texto. Se trata de pautas más que de reglas estrictas. Interesa más el contenido que el formato, así que no te sientas obligado a aprenderte la guía de estilos antes de colaborar. No te enojes ni te sorprendas si después un voluntario edita tu trabajo para que quede de acuerdo con esta guía.
+
+Los aspectos lingüísticos de esta guía se refieren principalmente a la documentación en idioma inglés.Se pueden (y se anima a) crear guías de estilo en otros idiomas. Estas deben publicarse como subpáginas de la página del equipo de localización.
+
+Si buscas especificaciones de cómo debe estructurarse un determinado tipo de página, mira la [guía de diseño de MDN](/es/docs/MDN/Contribute/Content/Layout).
+
+Aquí se enumeran los usos y costumbres recomendados a la hora de editar los artículos de este wiki. Si ves que falta contenido o crees que deberíamos mejorar o corregir algo, por favor coméntalo en la página de discusión.
+
+## Uso de mayúsculas en el nombre de las páginas y los encabezados
+
+- Cuando escribas los nombres de las páginas y de los encabezados, recuerda que sólo debe empezar con mayúscula la primera palabra de la frase.
+
+  - **Correcto**: "Guía de estilo"
+  - **Incorrecto**: "Guía de Estilo"
```

Comment:
In html this has color, I suppose this is not so important but just a comment

Reply:
Although color would be nice to differentiate Correct vs. Incorrect, color was removed upstream as well, since all `<font>` elements were to be removed.